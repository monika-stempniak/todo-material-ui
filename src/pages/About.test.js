import React from 'react';
import { render } from '@testing-library/react';

import { About } from '.';

describe('About component', () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<About />);
    expect(asFragment()).toMatchSnapshot();
  });
});
