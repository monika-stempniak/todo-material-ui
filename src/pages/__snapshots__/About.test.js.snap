// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root makeStyles-root-1 MuiContainer-fixed MuiContainer-maxWidthLg"
  >
    <h5
      class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
      data-testid="about-title"
    >
      About
    </h5>
    <p
      class="MuiTypography-root MuiTypography-body1"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis nulla expedita eum sequi incidunt vitae quisquam dicta beatae, quas corrupti blanditiis sed excepturi iure sit assumenda. Cupiditate pariatur repellendus veritatis!
    </p>
  </div>
</DocumentFragment>
`;
